irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -67,53 +74,59 @@ namespace {
 
   std::string checkAccountRolePermission(
       shared_model::interface::permissions::Role permission,
-      const std::string &account_alias = "role_account_id") {
+      const std::string &account_alias = ":role_account_id") {
     const auto perm_str =
         shared_model::interface::RolePermissionSet({permission}).toBitstring();
     const auto bits = shared_model::interface::RolePermissionSet::size();
     std::string query = (boost::format(R"(
           SELECT COALESCE(bit_or(rp.permission), '0'::bit(%1%))
```

Comment:
Is it possible to move all SQLs to separate files and load it into the runtime. In this case we will more flexible in terms of business logic right now.

Reply:
Added TODO